addons/l10n_ar_withholding/models/res_partner.py

```diff
@@ -0,0 +1,11 @@
+from odoo import models, fields
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    l10n_ar_tax_ids = fields.One2many(
```

Comment:
```suggestion
    l10n_ar_partner_tax_ids = fields.One2many(
```
Is not that clearer?

Reply:
I agree @jco-odoo . I changed it.